app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,35 @@
 class PostsController < ApplicationController
   def index
+    @posts = Post.all
     @user = User.find(params[:user_id])
-    @posts = Post.find_by(author_id: params[:id])
   end
 
   def show
-    @post = Post.find_by(id: params[:id])
+    # @user = User.find(params[:author_id])
+    # @post = Post.find_by(id:params[:id])
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻